### PR TITLE
Updated README to be 'sexier' and more informative

### PR DESCRIPTION
**Pull request by jef**
#### Database Migration

NO

#### Description

Current README is reletively bland and this README gives new users a little more information on where to find information for Sonarr.

image

### CODE REVIEW

**Comment by hijxf** [comment]
Anyone?

File: README.md [hunk 1]
```diff
@@ -1,58 +1,95 @@
...
+[![Docker](https://img.shields.io/badge/wiki-docker-1488C6.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+[![FAQ](https://img.shields.io/badge/wiki-FAQ-BF55EC.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/FAQ)
+
+- [Install Sonarr for your desired OS](https://github.com/Sonarr/Sonarr/wiki/Installation) or use [Docker](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+- Connect to the UI through http://localhost:8989 or http://your-ip:8989 in your web browser
+- See the [Wiki](https://github.com/Sonarr/Sonarr/wiki) for further configuration
+
+## Downloads
+
+Type | develop (stable) |
```
Comment (markus101) [inline README.md]: Develop and stable together doesn’t  really make sense to me. Releases on the release page are from master, not develop.

File: README.md [hunk 2]
```diff
@@ -1,58 +1,95 @@
...
+[![GitHub](https://img.shields.io/badge/github-issues-red.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/issues)
+[![GitHub Wiki](https://img.shields.io/badge/github-wiki-181717.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki)
+
+## Status
+
+[![GitHub issues](https://img.shields.io/github/issues/sonarr/sonarr.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/issues)
+[![GitHub pull requests](https://img.shields.io/github/issues-pr/sonarr/sonarr.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/pulls)
+[![Docker Pulls](https://img.shields.io/docker/pulls/linuxserver/sonarr.svg?maxAge=60&style=flat-square)](https://hub.docker.com/r/linuxserver/sonarr)
+[![GNU GPL v3](https://img.shields.io/badge/license-GNU%20GPL%20v3-blue.svg?maxAge=60&style=flat-square)](http://www.gnu.org/licenses/gpl.html)
+[![Copyright 2010-2017](https://img.shields.io/badge/copyright-2019-blue.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr)
```
Comment (markus101) [inline README.md]: Should be updated to 2019, though in general I’m not a fan of something that has to be updated every year, it’s a waste of everyone’s time.

File: README.md [hunk 3]
```diff
@@ -1,58 +1,95 @@
...
-* Fully configurable episode renaming
-* Full integration with SABnzbd and NZBGet
-* Full integration with Kodi, Plex (notification, library update, metadata)
-* Full support for specials and multi-episode releases
-* And a beautiful UI
+[![Installation](https://img.shields.io/badge/wiki-installation-brightgreen.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/Installation)
+[![Docker](https://img.shields.io/badge/wiki-docker-1488C6.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+[![FAQ](https://img.shields.io/badge/wiki-FAQ-BF55EC.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/FAQ)
+
+- [Install Sonarr for your desired OS](https://github.com/Sonarr/Sonarr/wiki/Installation) or use [Docker](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
```
Comment (markus101) [inline README.md]: Drop desired

File: README.md [hunk 4]
```diff
@@ -1,58 +1,95 @@
...
+Docker (linuxserver.io, x86_64, arm64, armhf) | [![Docker release](https://img.shields.io/badge/linuxserver-sonarr:latest-blue.svg?colorB=1488C6&maxAge=60&style=flat-square)](https://store.docker.com/community/images/linuxserver/sonarr)
+
+
+## Support
+
+[![Donate](https://img.shields.io/badge/donate-bitcoin-gold.svg?maxAge=60&style=flat-square)](https://sonarr.tv/donate)
+
+[![Discord](https://img.shields.io/badge/discord-chat-7289DA.svg?maxAge=60&style=flat-square)](https://discord.gg/M6BvZn5)
+[![Reddit](https://img.shields.io/badge/reddit-discussion-FF4500.svg?maxAge=60&style=flat-square)](https://www.reddit.com/r/sonarr)
+[![GitHub](https://img.shields.io/badge/github-issues-red.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/issues)
```
Comment (markus101) [inline README.md]: Issues are not for support, it should be dropped from here or the section renamed (also again issues is at the top of the page, I don’t see a reason to have it in the reader too).

File: README.md [hunk 5]
```diff
@@ -1,58 +1,95 @@
...
+[![GitHub Wiki](https://img.shields.io/badge/github-wiki-181717.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki)
+
+## Status
+
+[![GitHub issues](https://img.shields.io/github/issues/sonarr/sonarr.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/issues)
```
Comment (markus101) [inline README.md]: Issues and PRs are available on this page already, they don’t need to be called out here just to print the number of them.

File: README.md [hunk 6]
```diff
@@ -1,58 +1,95 @@
...
+[![GitHub pull requests](https://img.shields.io/github/issues-pr/sonarr/sonarr.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/pulls)
+[![Docker Pulls](https://img.shields.io/docker/pulls/linuxserver/sonarr.svg?maxAge=60&style=flat-square)](https://hub.docker.com/r/linuxserver/sonarr)
+[![GNU GPL v3](https://img.shields.io/badge/license-GNU%20GPL%20v3-blue.svg?maxAge=60&style=flat-square)](http://www.gnu.org/licenses/gpl.html)
```
Comment (markus101) [inline README.md]: badges are feeling over used, a simple link provides the same value.

**Comment by hijxf** [comment]
I think this would be more to your liking :)

**Comment by hijxf** [comment]
Looks a lot better now: https://github.com/Sonarr/Sonarr/blob/d4462ef5fe02fb7c8430688e136b3154b209cca5/README.md

File: README.md [hunk 7]
```diff
@@ -1,58 +1,78 @@
...
+- [Installation](https://github.com/Sonarr/Sonarr/wiki/Installation)
+- [Docker](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+- [FAQ](https://github.com/Sonarr/Sonarr/wiki/FAQ)
+- [Wiki](https://github.com/Sonarr/Sonarr/wiki)
+
+## Downloads
+
+Type | Releases |
+|---|---|
+Binary Releases | [GitHub Releases](https://github.com/Sonarr/Sonarr/releases)
```
Comment (markus101) [inline README.md]: Binary releases aren't linked here, our website has all the links though.

File: README.md [hunk 8]
```diff
@@ -1,58 +1,78 @@
...
+Docker (linuxserver.io, x86_64, arm64, armhf) | [Docker Releases](https://store.docker.com/community/images/linuxserver/sonarr)
+
+
+## Support
+
+- [Donate](https://sonarr.tv/donate)
+- [Discord](https://discord.gg/M6BvZn5)
+- [Reddit](https://www.reddit.com/r/sonarr)
+- [API v2](https://github.com/Sonarr/Sonarr/wiki/API)
```
Comment (markus101) [inline README.md]: I can't help but feel this doesn't fit, but not sure where it should go. Perhaps under getting started as `API Documentation`?

File: README.md [hunk 9]
```diff
@@ -1,58 +1,78 @@
...
+- Make sure `NzbDrone.Console` is set as the startup project
+- Build `NzbDrone.Windows` and `NzbDrone.Mono` projects
+- Build Solution
 
 ### UI Development
 
-* Run `yarn watch` to build UI and rebuild automatically when changes are detected
-* Run Sonarr.Console.exe (or debug in Visual Studio)
-
-### License
```
Comment (markus101) [inline README.md]: License should still be included.

**Comment by jef** [comment]
I tried to do some rebasing because of the crazy long commit history for this README, so I ended up making a new branch because of the merge hell. https://github.com/Sonarr/Sonarr/pull/3489

**Pull request closed without merging**